x-pack/plugins/observability_solution/synthetics/server/routes/overview_status/overview_status_service.ts

```diff
@@ -0,0 +1,331 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import moment from 'moment/moment';
+import { QueryDslQueryContainer } from '@elastic/elasticsearch/lib/api/typesWithBodyKey';
+import { SavedObjectsFindResult } from '@kbn/core-saved-objects-api-server';
+import { isEmpty } from 'lodash';
+import { withApmSpan } from '@kbn/apm-data-access-plugin/server/utils/with_apm_span';
+import { asMutableArray } from '../../../common/utils/as_mutable_array';
+import { getMonitorFilters, OverviewStatusQuery } from '../common';
+import {
+  getAllMonitors,
+  processMonitors,
+} from '../../saved_objects/synthetics_monitor/get_all_monitors';
+import { ConfigKey } from '../../../common/constants/monitor_management';
+import { RouteContext } from '../types';
+import {
+  EncryptedSyntheticsMonitorAttributes,
+  OverviewStatusMetaData,
+} from '../../../common/runtime_types';
+import { isStatusEnabled } from '../../../common/runtime_types/monitor_management/alert_config';
+import {
+  FINAL_SUMMARY_FILTER,
+  getRangeFilter,
+  getTimespanFilter,
+} from '../../../common/constants/client_defaults';
+
+type LocationStatus = Array<{
+  status: string;
+  locationId: string;
+  timestamp: string;
+}>;
+
+export const SUMMARIES_PAGE_SIZE = 5000;
+
+export class OverviewStatusService {
+  routeContext: RouteContext<Record<string, any>, OverviewStatusQuery>;
```

Comment:
```suggestion
```

This var is only referenced within the class and there's no need to define/assign it specifically in this manner.